src/base/static/components/organisms/map-legend-panel.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { Image } from "../atoms/imagery";
+import { Paragraph, Header4 } from "../atoms/typography";
+import MapLegendGroup from "../molecules/map-legend-group";
+
+import "./map-legend-panel.scss";
+
+const MapLegendPanel = props => {
+  return (
+    <div className="map-legend-panel">
+      {props.config.title && (
+        <Header4 classes="map-legend-panel__title">
+          {props.config.title}
+        </Header4>
+      )}
+      {props.config.description && (
+        <Paragraph classes="map-legend-panel__description">
+          {props.config.description}
+        </Paragraph>
+      )}
+      {props.config.groupings.map((grouping, i) => (
+        <MapLegendGroup key={i} {...grouping} />
```

Comment:
We should avoid using spread statements in our props as much as possible. It's much more clear to assign them explicitly:

```javascript
        <MapLegendGroup 
            key={i} 
            classes={grouping.classes} 
            content={grouping.content} 
            description={grouping.description} 
            title={grouping.title }
        />
```

The only reason we had to use the spread operator earlier was because we were prop drilling. But we resolved that issue by introducing redux, so we should no longer have to spread our props anymore.